python/lsst/sphgeom/pixelizations.i

```diff
@@ -41,7 +46,7 @@
             o = PyNumber_Multiply(tmp, tmp);
             Py_DECREF(tmp);
         } else {
-            if (p <= static_cast<unsigned long long>(PyInt_GetMax())) {
+            if (p <= static_cast<unsigned long long>(std::numeric_limits<long>::max())) {
                 o = PyInt_FromLong(static_cast<long>(p));
```

Comment:
I defer to you both - it seems like PyIntObject no longer exists in 3? I am OK with always using PyLongObject even on Python 2.